lib/neo4j/migration.rb

```diff
@@ -23,24 +23,30 @@ def joined_path(path)
     end
 
     class AddIdProperty < Neo4j::Migration
+      include Neo4j::Migrations::Helpers::IdProperty
+
       attr_reader :models_filename
 
       def initialize(path = default_path)
         @models_filename = File.join(joined_path(path), 'add_id_property.yml')
       end
 
       def migrate
+        ActiveSupport::Deprecation.warn '`AddIdProperty` task is deprecated and may be removed from future releases. '\
```

Comment:
I was actually thinking about putting the deprecation into 7.0.x and then removing it in 8.0.  What do you think?